packages/cli/src/cli.test.ts

```diff
@@ -44,13 +43,13 @@ describe('CLI tests', () => {
                 const outFilePath = path.join(outDir, `db.ts`);
 
                 // Check that file with the name doesn't exist yet
-                assert(!fs.existsSync(outFilePath));
+                expect(!fs.existsSync(outFilePath));
```

Comment:
@jimezesinachi pay more attention: you're not `expect`-ing enything 👀 

Also below